frontend/src/App.vue

```diff
@@ -15,10 +15,28 @@ import Menu from "@/components/menu/Menu.vue";
 import MainPage from "@/components/mainPage/MainPage.vue";
 import { container } from "jenesius-vue-modal";
 
+import { handleGoogleLogin, handleGoogleLogout, checkGoogleLogin } from "@/javascript/utils/googleLoginUtils";
+
 export default {
     // ERROR: Name "Menu" is reserved in HTML
     // eslint-disable-next-line
     components: { Menu, MainPage, Modal: container },
+
+    mounted() {
```

Comment:
To co wykonuje się w mounted przeniósłbym do funkcji np, initializeGoogle w googleLoginUtils.js a w mounted() tylko ją wywołał